internal/jq/jq_query.go

```diff
@@ -0,0 +1,206 @@
+package jq
```

Comment:
Nice! I think this should be under `pkg`, so we could use it from other projects if needed. If we are concerned about timing I'm ok merging as is and we can extract it in a follow up PR.

Reply:
Sure, I will move it.

Reply:
Done.